shoop/core/pricing/__init__.py

```diff
@@ -6,6 +6,9 @@
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
 from __future__ import unicode_literals
+import abc
+import six
+
 """
 Shoop modular product pricing functionality.
```

Comment:
Yeah, it needs updating. Also: Move it just below the copyright notice, before the imports. That's the correct location for module docstring.